src/common/util/addBgImage.js

```diff
@@ -0,0 +1,33 @@
+// import transformImage from "./transformImage";
+import getImageWidth from "./getImageWidth";
+
+const addBgImage = (
+  imageUrl,
+  gradientOverlay = "linear-gradient(to bottom, transparent, #181D1C)"
+) => {
+  let bgImage = {};
+  let imgSrc = "";
+
+  // Process image and set inline background image if image exists
+  if (imageUrl != null) {
+    if (imageUrl.startsWith("http")) {
+      const originalWidth = getImageWidth(imageUrl);
+
+      // Downsize image if it's wider than 2000px, otherwise just reduce jpg quality to 60%
+      if (originalWidth > 2000) {
+        imgSrc = imageUrl.concat("/2000x0");
+      } else {
+        imgSrc = imageUrl;
+      }
```

Comment:
For this repo, totally ok to take out image processing (resize, change jpg quality etc) since these were added to use with Storyblok's image service. For this repo, I think we can leave it up to the user to pass in the image processed on their end.